xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemuxFFmpeg.cpp

```diff
@@ -1678,7 +1679,7 @@ CDemuxStream* CDVDDemuxFFmpeg::AddStream(int streamIdx)
         }
 
         side_data = av_stream_get_side_data(pStream, AV_PKT_DATA_MASTERING_DISPLAY_METADATA, &size);
-        if (side_data && size)
+        if (side_data && size && (st->hdr_type == StreamHdrType::HDR_TYPE_NONE))
```

Comment:
This breaks side data handle on HDR10 streams because HDR type is set  to `StreamHdrType::HDR_TYPE_HDR10` due  `colorTransferCharacteristic == AVCOL_TRC_SMPTE2084`   in L1974